Samples/src/VisualDebugger/package.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
I'm under the impression that `package-lock.json` should not be committed, but I could be wrong.

Reply:
It looks like the npm documentation says it's intended to be committed as a snapshot of all the package versions in node_modules: https://docs.npmjs.com/files/package-lock.json